linux/vhba_hot_add_remove/vhba_device_hot_add_remove.yml

```diff
@@ -38,6 +38,32 @@
         - new_disk_ctrl_type == 'lsilogic'
         - "'Flatcar' not in guest_os_ansible_distribution"
 
+    - name: "Get file lsblk.sh for FreeBSD"
+      when: guest_os_ansible_distribution == "FreeBSD"
+      block:
+        - name: "Download script file lsblk.sh for FreeBSD"
+          ansible.builtin.get_url:
+            url: https://raw.githubusercontent.com/vermaden/scripts/master/lsblk.sh
+            dest: /tmp/lsblk.sh
+            mode: '0755'
+            timeout: 30
+            force: true
+          delegate_to: "{{ vm_guest_ip }}"
+          register: download_file_lsblk_result
+          ignore_errors: true
+        
+        - name: Display the result of file lsblk.sh downloading
+          ansible.builtin.debug: var=download_file_lsblk_result
+
+        - name: "Copy script file lsblk.sh into guest for FreeBSD"
+          ansible.builtin.copy:
+            src: ../utils/scripts/freebsd_lsblk.sh
+            dest: /tmp/lsblk.sh
+            mode: 0755
+            force: true
+          delegate_to: "{{ vm_guest_ip }}"
+          when: download_file_lsblk_result.status_code != 200
```

Comment:
This copy task can be removed. And do not ignore errors for above downloading task.